Druid/DRZ3ConstraintSolver.class.st

```diff
@@ -104,6 +104,17 @@ DRZ3ConstraintSolver >> isSatisfiable: aDRPiNode [
 	^ retval ifNil: [ true ]
 ]
 
+{ #category : 'as yet unclassified' }
+DRZ3ConstraintSolver >> isSatisfiable: aDRPiNode replacingPhi: phi for: phiOp [
+
+	| retval |
+	phiReplacement := phi -> phiOp.
+	retval := self isSatisfiable: aDRPiNode.
+	phiReplacement := nil.
+
+	^ retval
```

Comment:
This code looks duplicated now with the one in `DRConstantConstraintSolver`? Maybe a side effect of my merge?